### ISSUE TITLE
I get problem when i convert onnx model to openvino

### ISSUE THREAD

**Issue by khangnguyenhuu** [openvinotoolkit/openvino#9377]
[ ERROR ]  Exception occurred during running replacer "REPLACEMENT_ID" (<class 'extensions.load.onnx.loader.ONNXLoader'>): Unexpected exception happened during extracting attributes for node 3922.
Original exception message: The element type in the input tensor is not defined

I get this problem when i convert from onnx model to openvino
openvino version: 2021.4.752
Enviroment: openvino/ubuntu18_dev

My onnx model convert from torch

**Comment by Iffa-Meah** [comment]
Hi, could you provide more information on your model :
1. The topology of your model
2. Did you use any custom layer in your model, if yes, which layer it is?
3. Your MO conversion command



If possible, please share the trained model files as well.

**Comment by khangnguyenhuu** [comment]
> Hi, could you provide more information on your model :
> 
> 1. The topology of your model
> 2. Did you use any custom layer in your model, if yes, which layer it is?
> 3. Your MO conversion command
> 
> If possible, please share the trained model files as well.

Hi sir,
1. I have converted cascade mask rcnn with HRNet backbone implement in mmdetection, model achritecture can find here https://github.com/DevashishPrasad/CascadeTabNet
2. This framework is a blackbox with me and i dont know exactly whether they use any custom layer, but as my survey, i think dont have any custom layer was used in this model
3. My MO command: python3 mo.py --input_model my_model.onnx

I cant share my trained model, but you can find pretrained model here https://drive.google.com/file/d/1-QieHkR1Q7CXuBu4fp3rYrvDG9j26eFT/view

**Comment by Iffa-Meah** [comment]
I've cross-checked your model topology and it seems that it is not supported. Neither Cascade MaskRCNN nor HRNet is listed in this [official documentation.](https://docs.openvino.ai/latest/openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_ONNX.html) Hence, issues are expected.

It would be great if you could provide the step/script that you had used to convert your .pth file into .onnx so that I could try it from my side and double confirm the IR conversion result.

**Comment by khangnguyenhuu** [comment]
i have cloned & setup this repository: https://github.com/DevashishPrasad/CascadeTabNet
it will auto clone mmdetection as sub-repository to run and i use tool in mmdetection to convert from .pth file to .onnx file 
my convert command: python3 mmdetection/tools/conver2onnx.py <config_path> <checkpoint_path> --output_file cascade.onnx

**Comment by Iffa-Meah** [comment]
@khangnguyenhuu  thank you for being patient. I did some digging into the repos that you shared.

I cloned both "https://github.com/DevashishPrasad/CascadeTabNet.git"  and "https://github.com/open-mmlab/mmdetection.git".

![tools](https://user-images.githubusercontent.com/79643476/148490468-7c49ca85-c299-4017-ba4a-f1bba6176160.png)

However, the only conversion script that I found was the [script](https://github.com/DevashishPrasad/CascadeTabNet/blob/master/Data%20Preparation/generateVOC2JSON.py) to convert Pascal VOC XML annotation files to a single COCO Json file.

Your conversion script named "conver2onnx.py" seems to be originated from somewhere else. If you share this script or its source, it would definitely help to double-check your model's issue.
